src/Avalonia.Interactivity/RoutedEvent.cs

```diff
@@ -113,24 +113,38 @@ public RoutedEvent(string name, RoutingStrategies routingStrategies, Type ownerT
             Contract.Requires<ArgumentNullException>(ownerType != null);
         }
 
+        [Obsolete("Use overload taking Action<TTarget, TEventArgs>.")]
```

Comment:
I don't know if we want to mark this obsolete but I don't really feel super strongly about it.

Reply:
I originally thought about removing this completely as I see no value in this method, but I thought that somebody might be using it so I made it obsolete. Do you think that I should leave this in non-obsolete state?